tests/integrations/Cargo.toml

```
@@ -8,7 +8,7 @@ license.workspace = true
 workspace = true
 
 [dependencies]
-alloy = { version = "0.6.4", features = ["full"] }
+alloy = { version = "0.8.1", features = ["full"] }
```

Would the following work or does it cause trouble?
```suggestion
alloy.workspace = true
```